pressbooks/pressbooks
# feat: Add 'Institution' field to book info

Closes #2407 

This PR adds a new optional multi select `Institutions` field to the book info page. It also exposes the new field to the `wp-json/pressbooks/v2/metadata` endpoint so the cloner is able clone the new information.

#### How to test

1. Make sure the new 'Institution(s)' field gets displayed in the Book Info section
2. Make sure you're able to save institutions
3. Make sure values are exposed properly (books with no institutions should not expose the information)
4. Make sure cloning routine works without issues and the institutions are cloned properly
